edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -692,6 +707,30 @@ impl Authenticator for DockerModuleRuntime {
     }
 }
 
+fn init_client(docker_url: &Url) -> Result<DockerClient<UrlConnector>> {
```

Comment:
Don't generate a new `hyper::Client` for every `ModuleRuntime` invocation. There should be only one per `ModuleRuntime` like there was before.